dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Grid/LoadFlow.mo

```diff
@@ -0,0 +1,807 @@
+within Dynawo.Examples.Nordic.Grid;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model LoadFlow "Nordic test grid with buses, lines, shunts, constant loads, transformers and generators"
+  import Modelica.ComplexMath;
+  import Modelica.SIunits;
+  import Dynawo;
+  import Dynawo.Electrical;
+  import Dynawo.Electrical.SystemBase;
+  import Dynawo.Types;
+
+  extends Dynawo.Examples.Nordic.Grid.BaseClasses.NetworkWithPQLoads;
```

Comment:
Maybe write a comment or prepare a commented line to say it could also extend AlphaBeta Network